deploy/scripts/000_deploy_test_stubs.ts

```diff
@@ -3,54 +3,57 @@ import { DeployFunction } from "hardhat-deploy/types";
 import { ERC20_DECIMALS, ETH_PRICE_IN_USD, TST1_PRICE_IN_USD } from "../../test/Constants";
 import { BigNumber } from "ethers";
 
-const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, getNamedAccounts } = hre;
+const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId } = hre;
   const { deploy } = deployments;
 
   const { deployer } = await getNamedAccounts();
 
-  await deploy("TestOracleEth", {
-    contract: "TestOracle",
-    from: deployer,
-    args: [ETH_PRICE_IN_USD],
-    log: true,
-  });
-
-  await deploy("TestOracleTst1", {
-    contract: "TestOracle",
-    from: deployer,
-    args: [TST1_PRICE_IN_USD],
-    log: true,
-  });
-
-  const startingSupply = BigNumber.from("1000000").mul(ERC20_DECIMALS);
-  await deploy("TestToken1", {
-    contract: "TestToken",
-    from: deployer,
-    args: [startingSupply, "Test1", "TST1"],
-    log: true,
-  });
-
-  await deploy("TestToken2", {
-    contract: "TestToken",
-    from: deployer,
-    args: [startingSupply, "Test2", "TST2"],
-    log: true,
-  });
-
-  const WETH = await deploy("WETH", {
-    contract: "TestToken",
-    from: deployer,
-    args: [startingSupply, "WETH", "WETH"],
-    log: true,
-  });
-
-  await deploy("TestSwapRouter", {
-    contract: "TestSwapRouter",
-    from: deployer,
-    args: [WETH.address],
-    log: true,
-  });
+  // we only need TestStubs when running tests
+  if ((await getChainId()) == "31337") {
```

Comment:
chain id is always this no matter how you are running the test? unit test, various EVM chain testnets? Even so, this should be a constant somewhere

Reply:
This assumes tests are run against `localhost`. 
If you're running against testnets, shouldn't we use testnet addresses instead of deploying our own testWETH?

Reply:
More fundamentally, shouldn't we be testing on a fork of the mainnet and not running tests on live nets?

Reply:
yeah makes sense. need to look around at the tooling available and decide what the most practical solution is

Reply:
Resolving.